kasa/iot/iotmodule.py

```diff
@@ -1,5 +1,7 @@
 """Base class for IOT module implementations."""
 
+from __future__ import annotations
+
```

Comment:
Does it make any difference to add this into files where it's no concern at the moment? Keeping it out would keep these files and this PR diff cleaner, assuming they are not injected automatically by a ruff rule?

Reply:
They were not injected automatically, I had to do that myself with a script.  I thought it was simpler to just have it every files so any new contributors do not have to work it out for themselves. I can remove them if you prefer.

Reply:
Actually ruff will tell them about it anyway so will remove the unnecessary ones